### ISSUE TITLE
Milestones

### ISSUE THREAD

**Issue by rspds** [cajubavictor/snake-killator#16]
Iremos adicionar milestones? Quais datas?

**Comment by cajubavictor** [comment]
Sim,seria uma boa ideia para nos organizarr,  o ideal seria um tempo de pelo menos 5 dias entre uma e outra

**Comment by cajubavictor** [comment]
ja as criei , se quiserem alguma alteraçao...
